company/about-mattermost/business-model.md

```diff
@@ -1,22 +1,22 @@
+The paid enterprise edition of Mattermost is built for **team managers** and **IT professionals** as a flexible, high trust messaging platform that meets **compliance** needs in serving multiple teams of developers and end users in high security environments.
 
 ## Frequently asked questions
 
-### What keeps other companies from using your open source software to create offerings that compete with your paid products? 
+### What keeps other companies from using your open source software to create offerings that compete with your paid products?
 
-We use the acronym "TACTICAL" to summarize the approach to maintaining differentiation of the paid version of our offerings to offerings a competitor may create from our open source code base:    
+We use the acronym "TACTICAL" to summarize the approach to maintaining differentiation of the paid version of our offerings to offerings a competitor may create from our open source code base:
   
-* **Test infrastructure:** We use [SaaS QA](https://www.rainforestqa.com/) which isn't included in the open source version 
-* **Alignment:** Developer features are free. Paid features are for managers and IT pros, who are less likely to spend time and effort forking and maintaining an open source project vs. deploying budgets to purchase paid features
-* **Core committers:** We work with an active community of [core committers](https://developers.mattermost.com/contribute/getting-started/core-committers/) to keep the project and business in sync
-* **Threat intelligence:** Mattermost's [responsible disclosure policy](https://mattermost.com/security-vulnerability-report) means forks will lag in threat intelligence
-* **Innovation:** Monthly ship cycles and high velocity innovation rapidly lower the value of forks
-* **Compatibility:** Only genuine versions upgrade smoothly to new releases and security patches 
-* **Architecture:** Modular architecture makes the need to fork rare
-* **Licensing:** Mattermost's trademark is registered internationally, forking requires renaming
+* **Test infrastructure:** We use [SaaS QA](https://www.rainforestqa.com/) which isn't included in the open source version.
+* **Alignment:** Developer features are free. Paid features are for managers and IT pros, who are less likely to spend time and effort forking and maintaining an open source project vs. deploying budgets to purchase paid features.
+* **Core committers:** We work with an active community of [core committers](https://developers.mattermost.com/contribute/getting-started/core-committers/) to keep the project and business in sync.
+* **Threat intelligence:** Mattermost's [responsible disclosure policy](https://mattermost.com/security-vulnerability-report) means forks will lag in threat intelligence.
+* **Innovation:** Continuous ship cycles and high velocity innovation rapidly lower the value of forks.
```

Comment:
I think we can say "regular". We're moving towards "continuous delivery" but it's not quite there yet.